ref/android-features.md

```diff
@@ -0,0 +1,21 @@
+Anbox Cloud implements support for various features Android has. The following provides an overview which features are currently avilable as part of Anbox Cloud. If a feature is not listed, it is currently not supported.
+
+| Feature           | Supported | Notes  |
+|--------------------|:-----------------:|
```

Comment:
```suggestion
|--------------------|:-----------------:|-------------|
```
(Not sure if the number of characters is correct ;) - but we need another column)

Reply:
The number of characters shouldn't matter, I stopped being precise here :-)